docs/README.md

```diff
@@ -2,13 +2,73 @@
 
 ## Welcome to the BanyanDB
 
-Here you can learn all you need to know about BanyanDB. 
+**BanyanDB**, as an observability database, aims to ingest, analyze and store Metrics, Tracing, and Logging data. It's designed to handle observability data generated by **Apache SkyWalking**. Before BanyanDB emerges, the Databases that SkyWalking adopted are not ideal for the APM data model, especially for saving tracing and logging data. Consequently, There’s room to improve the performance and resource usage based on the nature of SkyWalking data patterns.
 
-- **Installation**. Instruments about how to download and onboard BanyanDB server, Banyand.
-- **Clients**. Some native clients to access Banyand.
-- **Observability**. Learn how to effectively monitor, diagnose and optimize Banyand.
-- **Concept**. Learn the concepts of Banyand. Includes the architecture, data model, and so on.
-- **CRUD Operations**. To create, read, update, and delete data points or entities on resources in the schema.
+Here you can learn all you need to know about BanyanDB. Let's get started with it.
+
+### Introduction
```

Comment:
I think all following are duplicate with menu file?

Reply:
Yes, may be more beginner-friendly, README page now is relatively simple.

Reply:
I think if you want to make that more clear, you should enhance the menu. 
Rather than duplicate the data. Readme shows up here https://skywalking.apache.org/docs/skywalking-banyandb/latest/readme/, too long page seems not friendly.

Reply:
The current menu, with the exception of Concept -> Concepts, doesn't need much to optimize. 

We may need to enrich each chapter, and then subdivide the menu items. 

Regarding the `Installation` menu,  although it is not in the same page to complete the Installation, you need to Get binaries+standalone mode/Get binaries+cluster mode to finish the deployment, that's ok. it is also possible, the content and the steps can be rich.

Reply:
I'll try to enrich the content of the page again, and not change README and the menu.